optics-core/src/Optics/Label.hs

```diff
@@ -87,9 +106,196 @@
 --
 -- >>> peter & set #pets []
 -- Human {humanName = "Peter", humanAge = 13, humanPets = []}
+
+-- $guide
+--
+-- == The problem
+--
+-- Standard Haskell records are a common source of frustration amongst seasoned
+-- Haskell programmers. Their main issues are:
+--
+-- 1) Pollution of global namespace as every field accessor is also a top-level
+--    function.
+--
+-- 2) Inability to define multiple data types sharing field names in the same
+--    module.
+--
+-- 3) Clunky update syntax, especially when nested fields get involved.
```

Comment:
Perhaps you could reorder this list so it matches the order in which the points are referenced below?

Reply:
Makes sense :+1: Done.